packages/client/hmi-client/src/components/widgets/tera-resizable-panel.vue

```diff
@@ -0,0 +1,71 @@
+<!--
+TeraResizablePanel: AVertically resizable panel with a handle along the bottom
+-->
+
```

Comment:
We usually put those comments in the `<script>` as classic multiline comment `/** */`